.env.example

```
@@ -10,4 +10,11 @@ GOOGLE_CLIENT_ID="your-google-client-id"
 GOOGLE_CLIENT_SECRET="your-google-client-secret"
 
 GITHUB_CLIENT_ID="your-github-client-id"
-GITHUB_CLIENT_SECRET="your-github-client-secret"
\ No newline at end of file
+GITHUB_CLIENT_SECRET="your-github-client-secret"
+
+#using gmail
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=587
+SMTP_USER=yourname@gmail.com
+SMTP_PASS=your_app_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMTP_PASS key should go before the SMTP_PORT key

(UnorderedKey)